share/lib/python/neuron/__init__.py

```diff
@@ -158,6 +158,7 @@
 h = hoc.HocObject()
 version = h.nrnversion(5)
 __version__ = version
+_userrxd = 0
```

Comment:
Is this flag 0 or 1 instead of False or True because of HOC?

Reply:
@sgamiye : could you check the above comment ?

Reply:
Fixed in latest version.